_episodes/arrays.md

```diff
@@ -421,3 +421,16 @@ which returns a random sample of the values in the list.
 > >
 > {: .solution }
 {: .challenge }
+
+> ## Comments on a static blog
+>
+> Comments section can be included in a static blog using one of the below approaches.
+> 1. Disqus is the first approach. Disqus website offers [instructions](https://disqus.com/admin/install/platforms/jekyll/) on how to enable comments for Jekyll platform (used for GitHub pages). [Demo](https://erresen.github.io/jekyll/disqus/2016/08/26/getting-comments-working-jekyll-disqus.html). Although if you are privacy inclined, the likes of Privacy Badger blocks Disqus widget and might look for alternatives.
```

Comment:
```suggestion
> 1. Disqus is the first approach. The Disqus website offers 
>    [instructions on how to enable comments for Jekyll platform (used for GitHub pages)](https://disqus.com/admin/install/platforms/jekyll/). 
>    [Here is a demo of Disqus with Jekyll](https://erresen.github.io/jekyll/disqus/2016/08/26/getting-comments-working-jekyll-disqus.html). 
>    Although if you are privacy inclined, browser extensions such as Privacy Badger block the Disqus widget so you might look for alternatives.
```

Split source text across multiple lines, and expand the description a little bit e.g. many readers may not know what Privacy Badger is.

Reply:
I have expanded the link text in each of these points, [to make the links more accessible e.g. for users visiting the site with a screen reader](https://www.a11yproject.com/posts/2019-02-15-creating-valid-and-accessible-links/#toc_Write-helpful-link-text).

Reply:
agreed Toby, great suggestion!